### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by mmanciop**

### CODE REVIEW

**Review by sed-i — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+- Generally, before developing enhancements to this charm, you should consider [opening an issue](https://github.com/canonical/lma-light-bundle) explaining your use case.
+- If you would like to chat with us about your use-cases or proposed implementation, you can reach us at [Canonical Mattermost public channel](https://chat.charmhub.io/charmhub/channels/charm-dev) or [Discourse](https://discourse.charmhub.io/).
+- All enhancements require review before being merged.
+  Besides the code quality and test coverage, the review will also take into  account the resulting user experience for Juju administrators using this charm.
+  Please help us out in having easier reviews by rebasing onto the `main` branch, avoid merge commits and enjoy a linear Git history.
+
+## Development
+
+### Deploy with local bundle file
```
Comment (jnsgruk) [inline CONTRIBUTING.md:13]:
```suggestion
### Deploy from a local bundle file
```

File: README.md [hunk 2]
```diff
@@ -1,10 +1,29 @@
-# LMA light bundle
-This juju bundle deploys a light LMA stack with the following interrelated charms:
-- [grafana](https://charmhub.io/grafana-k8s) ([source](https://github.com/canonical/grafana-operator))
-- [prometheus](https://charmhub.io/prometheus-k8s) ([source](https://github.com/canonical/prometheus-operator))
-- [alertmanager](https://charmhub.io/alertmanager-k8s) ([source](https://github.com/canonical/alertmanager-operator))
+# LMA Light bundle
+
+At Canonical, LMA stands for "Logs, Metrics and Alerts", and is a moniker to describe observability stacks.
```
Comment (jnsgruk) [inline README.md:3]: I thought it was `Logging, Monitoring and Alerting`?

File: README.md [hunk 3]
```diff
@@ -1,10 +1,29 @@
...
+- [Grafana](https://charmhub.io/grafana-k8s) ([source](https://github.com/canonical/grafana-operator))
+- [Prometheus](https://charmhub.io/prometheus-k8s) ([source](https://github.com/canonical/prometheus-operator))
+- [Alertmanager](https://charmhub.io/alertmanager-k8s) ([source](https://github.com/canonical/alertmanager-operator))
+
+This bundle is very much work in progress.
+For example, there is currently no "L" in LMA, i.e., there is no logging component integrated yet, there are several administrative use-cases to support, and more.
+
+## The Vision
+
+LMA Light is going to be the go-to solution for monitoring Canonical appliance when the end user does not already have an established observability stack.
```
Comment (jnsgruk) [inline README.md:15]:
```suggestion
LMA Light is going to be the go-to solution for monitoring Canonical appliances when the end user does not already have an established observability stack.
```

File: README.md (hunk 2 above)
Comment (mmanciop) [reply README.md:3]: Gods below I hope not. Implying that “monitoring” is just about metrics is abhorrent.

**Review by sed-i — APPROVED** [review]

**Review by jnsgruk — APPROVED** [review]

**Pull request merged**